AUTHORS

```diff
@@ -39,4 +39,3 @@ Dean Elbaz <elbaz.dean@gmail.com>
 Mike Berman <evencode@gmail.com>
 Dmitriy Fedorenko <c0va23@gmail.com>
 Zach Marcantel <zmarcantel@gmail.com>
-James Maloney <jamessagan@gmail.com>
```

Comment:
Right, so you're talking about this test failure:

```
=== RUN TestMarshal
--- FAIL: TestMarshal (0.00 seconds)
panic: value method gocql.CustomString.MarshalCQL called using nil *CustomString pointer [recovered]
```

I can see what you are trying to achieve and in your favor you have written a covering test case, but considering that this is very critical to the library, I think this change requires a second opinion (@phillipCouto).

We need to be sure what the best behavior is.